components/Header/header.js

```diff
@@ -1,72 +1,120 @@
+/* eslint-disable @next/next/no-img-element */
 /* eslint-disable react/no-unescaped-entities */
 import React from "react";
-import Scrollspy from "react-scrollspy";
-import Pattern1 from "../illustrations/pattern1";
-import Pattern2 from "../illustrations/pattern2";
+import Button from "../Button/button";
+import Logo from "../illustrations/logo";
+import Stroke1 from "../illustrations/stroke1";
+import Stroke2 from "../illustrations/stroke2";
+import Step from "../illustrations/stepLike";
+
+const data = [
+  {
+    title: "What is the AsyncAPI Conference?",
+    description:
+      "At the 2022 AsyncAPI Conference, speakers from all around the world will join us online to present how they are using the AsyncAPI Specification in their real-world use cases, how they use tooling to support the specification, and how they have contributed to the project.",
+    button: (
+      <Button
+        text="Visit the AsyncAPI Website"
+        link="https://www.asyncapi.com/"
+        className="text-[21px] py-[14px] px-[26px] mt-[32px]"
+      />
+    ),
+  },
+  {
+    title: "Apply to be a sponsor",
+    description:
+      "We are in the process of looking for those interested in sponsoring the upcoming AsyncAPI Conference.",
+    button: (
+      <Button
+        text="Become a sponsor"
+        link="https://github.com/asyncapi/community/files/9568764/Sponsorship_Prospectus_2022_v2.pdf"
```

Comment:
I think we could just have a button on the website somewhere that says "Download Prospectus" and is just a download link

Reply:
you mean `Become a sponsor` is not a good button name?

Reply:
Oh i see, I wasn't thinking when I wrote that I guess haha 😅 

Yeah this works! As long as it will open in a new tab and not take you away from the website. Disregard what I said about making it a download link as well